### ISSUE TITLE
No option to enable/disable Touch to Search in settings

### ISSUE THREAD

**Issue by cbwee** [brave/brave-browser#17252]
## Description 
When Brave asked me if I wanted to enable or disable Touch to Search, I chose to disable it, but when I wanted to enable it again, I went to the settings in the browser and there was no option to enable it again.

## Steps to reproduce 
   1. Select any text on a webpage.
   2. Touch to Search appears, and it asks if you want to keep it enabled or disable it.
   3. Choose disable.
   4. Try to enable Touch to Search in the settings.
   5. The option cannot be found in the settings.

## Issue reproduces how often 
Easily reproduced

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? 
Yes
- Can you reproduce this issue with the current Play Store Beta version? 
Yes
- Can you reproduce this issue with the current Play Store Nightly version?
Yes

## Device details
   - Install type (ARM, x86): arm64-v8a
   - Device type (Phone, Tablet, Phablet): Phone (Xiaomi POCO M3)
   - Android version: Android 10
   
## Brave version
Stable 1.27.109
Beta 1.28.96
Nightly  1.29.38

**Comment by vrindavan** [comment]
Waiting for this!

**Comment by mattia-b89** [comment]
with version 1.51.118 same problem;
here is a screenshot:  

<img src="https://github.com/brave/brave-browser/assets/545463/997bd763-e4a1-4777-9df1-e44b35f01c78" alt="drawing" width="200"/>  

steps to reproduce:  
1) tap a word, 'Performance' in this case (RED ARROW)
2) a bottom bar will appear with a Google search results (GREEN ARROW)

NB: touch to search "works" only when **Google** is the selected search engine.

**Comment by pgorod** [comment]
Can somebody please **add an Accessibility label** to this one? I can't tell you how cumbersome it is to have these super-annoying pop-ups show up all the time, with no way to disable them, if you have visual difficulties or movement difficulties. It's like an extra hurdle on each time one taps a link. And yes, some sites have something about their design (I think it's some kind of JS buttons instead of regular html links) that makes this pop-up in all navigation, when it really shouldn't.

It's totally shameful to have this sort of intrusive annoyance be impossible to turn off. For years now!

If there is some workaround, some under-the-hood option that can be tweaked, please tell us now...

**Comment by BobT36** [comment]
Yes it's VERY annoying when you just can't click certain buttons because of the stupid popups.

**Comment by ryanbr** [comment]
Is this specific flag still current @schipht ?

Looking at the Chrome 133; 

![Image](https://github.com/user-attachments/assets/88af3514-f724-4f98-b116-e30d1a0dacc6)

**Comment by pgorod** [comment]
I had to stop using Brave because of this. 

In my last post here I asked for an accessibility label to be added to this issue and provided a rationale. Can somebody please take a look at that? Thanks

**Comment by Cade-H** [comment]
I just recently started using Brave on Android and was trying to find a way to disable this feature. Come to find out you can't! This is making me somewhat regret switching. Pretty wild that this issue has been open since 2021 without resolve. How hard is it to add a simple toggle to the settings?

**Comment by bsclifton** [comment]
cc: @timchilds

**Comment by timchilds** [comment]
@SergeyZhukovsky per triage, let's bring the ON/OFF setting over to `Search Engine`s section. Only show when Google is the default SE for either standard or private.

**Comment by Arutemu64** [comment]
Will this ever be fixed? That's a massive privacy issue, I don't want everything I (accidentally) select being sent to Google Search.

**Comment by andys8** [comment]
I think I found a way to disable the feature. Please confirm.

1. Delete all data including cache for Brave mobile
2. Switch search engine to Google
3. Trigger "touch to search" on a random website
4. The first time there is dialog: Click the tiny text "settings"
5. Disable "Touch to search" in the settings

![Image](https://github.com/user-attachments/assets/0793e7f4-b956-4e8d-9e09-626f6eeb5888)
![Image](https://github.com/user-attachments/assets/3b6fd216-ab66-4a7b-9e39-4273cf191cfd)

**Comment by rotekdev** [comment]
> [@SergeyZhukovsky](https://github.com/SergeyZhukovsky) per triage, let's bring the ON/OFF setting over to `Search Engine`s section. Only show when Google is the default SE for either standard or private.

@timchilds 
Why not just enable it for other search engines? Recently i tried to change search engine from Google to Ecosia (with Tap to search being already turned on), and it worked, until i shut down the browser (probably then it was filtered back only to google).

It would benefit also the users of brave search engine.
